CONTRIBUTING.md

```diff
@@ -1,7 +1,50 @@
 OpenProject is an open source project and we encourage you to help us out. We'd be happy if you do one of these things:
 
-* Create a [new work package on openproject.org](https://community.openproject.org/projects/openproject/work_packages) if you find a bug or need a feature
-* Help out other people on our [forums](https://community.openproject.org/projects/openproject/boards)
-* Help us [translate OpenProject to more languages](https://community.openproject.org/projects/openproject/wiki/Translations)
-* Contribute code via GitHub Pull Requests, see our [contribution page](https://community.openproject.org/projects/openproject/wiki/Contribution) for more information
-* See [Git Flow](https://community.openproject.org/projects/openproject/wiki/Git_Branching_Model)
+## Issue flow
+
+Ideally, there is a work package for every issue that is tackled by a contributor - the list can be found [here](https://community.openproject.org/projects/openproject/work_packages). Work packages can be created there on-demand if necessary.
+
+## Development flow
+For contributing source code, please follow the Git Workflow below:
+
+- Fork Openproject on GitHub
+- Clone your fork to your development machine: 
+
+```
+git clone git@github.com/<username>/openproject
+```
+
+- Optional: Add the original OpenProject repository as a remote, so you can fetch changes: 
+
+```
+git remote add upstream git@github.com:/opf/openproject
+```
+
+- Make sure you're on the right branch, right now (February 2015), the main development branch is dev: 
```

Comment:
Also: What if I have something from what I believe should be included in a backport branch?

Is this possible at all? What should I do, which limitations are there? How is this backport branch called?